paper-checkbox.html

```diff
@@ -43,18 +43,17 @@
 @group Paper Elements
 @class paper-checkbox
 -->
+<style is="custom-style">
+  * {
+    --paper-checkbox-unchecked-color: var(--primary-text-color);
+    --paper-checkbox-unchecked-ink-color: var(--primary-text-color);
 
-<dom-module id="paper-checkbox">
-  <style is="custom-style">
-    * {
-      --paper-checkbox-unchecked-color: var(--primary-text-color);
-      --paper-checkbox-unchecked-ink-color: var(--primary-text-color);
-
-      --paper-checkbox-checked-color: var(--default-primary-color);
-      --paper-checkbox-checked-ink-color: var(--default-primary-color);
-    }
-  </style>
+    --paper-checkbox-checked-color: var(--default-primary-color);
+    --paper-checkbox-checked-ink-color: var(--default-primary-color);
+  }
+</style>
```

Comment:
Yesterday we decided to do the element defaults in separate PRs since it's not a broken API. I can go either way.